ui/src/app/cohort-search/attributes-page/attributes-page.component.spec.tsx

```diff
@@ -1,7 +1,3 @@
-import { mount, ReactWrapper, ShallowWrapper } from 'enzyme';
```

Comment:
TBH I prefer separating them in this way

Reply:
BUT I prefer the linter making this decision for me more.

Reply:
Yeah, not a big deal but I do prefer them separated as well. [Their example](https://github.com/lydell/eslint-plugin-simple-import-sort/#example) shows the packages separated from the other imports. I see they also have a [grouping option](https://github.com/lydell/eslint-plugin-simple-import-sort/#custom-grouping) with some default options listed. Should we be getting those default groups here or do we need to configure that?

Reply:
But I am good with the current changes as well. I agree I would rather have the linter take care of it than spend time on custom config for the grouping.

Reply:
Good find.  `grouping` looks simple and powerful enough to get a result I like.  I'll update the PR with a grouping proposal.